paymentsheet/src/main/java/com/stripe/android/paymentsheet/address/TransformAddressToElement.kt

```diff
@@ -132,6 +132,11 @@ internal fun parseAddressesSchema(inputStream: InputStream?) =
         )
     }
 
+internal fun parseKlarnaCurrencyMap(inputStream: InputStream?) =
```

Comment:
This implementation doesn't seem to be bound to Klarna? could we rename it to a more generic name?